packit/cli/init.py

```diff
@@ -117,9 +117,17 @@ def init(
             "Generating source-git repositories is experimental, "
             "please give us feedback if it does things differently than you expect."
         )
-        api = get_packit_api(
-            config=config, local_project=path_or_url, load_packit_yaml=False
-        )
+        try:
+            api = get_packit_api(
+                config=config, local_project=path_or_url, load_packit_yaml=False
+            )
+        except PackitException:
+            logger.error(
+                "The init command is expected to be run in a git repo "
+                "which will turn into a source-git repo. We suggest to run the command "
+                "in a blank git repository or in a nre branch of the upstream project."
+            )
```

Comment:
Does the error above guarantee that the error raised is always the one this message talks about? What if `get_packit_api()` raises a PackitException for some other reason now or in the future? That would make this message misleading.

Reply:
that's a good point: I'll rework it so that we raise "not a git repo" exc and will catch it here